primehub/extras/templates/examples/deployments.md

```diff
@@ -1,3 +1,18 @@
+### Fields for creating or updating
+
+| field | required | type | description |
+| --- | --- | --- | --- |
+| name | required | string | the name of the deployment |
+| id | optional | string | either self-defined or system generated automatically if not specified |
```

Comment:
Auto-filling feature should not change a field required or not. Please refer to other review comments